module/importer-magento-2/src/Infrastructure/Generator/Magento2TransformerGenerator.php

```diff
@@ -12,13 +12,12 @@
 use Ergonode\SharedKernel\Domain\Aggregate\TransformerId;
 use Ergonode\Transformer\Domain\Entity\Transformer;
 use Ergonode\Transformer\Infrastructure\Converter\TextConverter;
-use Ergonode\Attribute\Domain\Repository\AttributeRepositoryInterface;
 use Ergonode\ImporterMagento2\Domain\Entity\Magento2CsvSource;
 use Ergonode\Transformer\Infrastructure\Generator\TransformerGeneratorStrategyInterface;
 
 class Magento2TransformerGenerator implements TransformerGeneratorStrategyInterface
 {
-    private AttributeRepositoryInterface $repository;
```

Comment:
It would be good to check why it is commented on in the code before removing it.

Reply:
The code is commented out on "Add Import maching system and prcoes refactoring" - tbh it does not tell me much so I commented out the repository(nor removed) for internal consistency but I'm not really able to do anything more - I could remove the commented code itself